### ISSUE TITLE
Error 501 when use SolrCloud

### ISSUE THREAD

**Issue by diegobrum** [solr-express/solr-express#203]
**Environment**
*Solr version:* 6.4.1
*Mode:* Cloud

**What?**

Error in connection with Solr.

```
The remote server returned an error: (501) Method GET-X is not defined in RFC 2068 and is not supported by the Servlet API
```

**Comment by diegobrum** [comment]
Closes #199

**Comment by diegobrum** [comment]
Works correctly in .Net Core
